src/structures/messageCreate.js

```diff
@@ -12,15 +12,20 @@ client.on('messageCreate', async (message) => {
     if (afkjson.afkvalue.indexOf(message.author.id) != -1) {
         afkjson.afkvalue.splice(afkjson.afkvalue.indexOf(message.author.id), 1);
         fs.writeFileSync('./src/data/afk.json', JSON.stringify(afkjson));
-        message.member.setNickname(message.author.username)
+        message.member.setNickname(message.author.username);
         message.channel.send(`**\`${message.author.username}\` telah kembali dari AFK!**`);
     }
 
-    let mentioned = message.mentions.members.first();
+    let mentioned = message.mentions.members?.first();
     
     if (mentioned) {
-        if (afkjson.afkvalue.indexOf(mentioned.id) != -1)
-        message.channel.send(`**\`${mentioned.user.username}\` sedang AFK!**`);
+        if (afkjson.afkvalue.indexOf(mentioned.id) != -1) {
+            return message.channel.send(`**\`${mentioned.user.username}\` sedang AFK!**`);
+        } else {
+            return;
+        }
+    } else if (mentioned == null) {
```

Comment:
isn't that supposed to be undefined?

Reply:
`first()` method throw an error "Unhandled Promise Rejection: TypeError: Cannot read properties of null" which will give a null output, it would be much safer to use null as a comparison.

Reply:
```suggestion
    } else if (mentioned == undefined) {
```
but since you have optional chaining, it only results in undefined right? it would be more proper to use undefined IMO
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Since null is loose equal to undefined it wouldn't be a problem to use that.

Reply:
understood!